RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -251,7 +251,7 @@ boolean auto_canUseJuneCleaver() {
 
 boolean auto_juneCleaverAdventure()
 {
-	if (!auto_canUseJuneCleaver() || get_property("_juneCleaverFightsLeft").to_int() > 0)
+	if (!auto_canUseJuneCleaver() || get_property("_juneCleaverFightsLeft").to_int() > 0 || !zone_isAvailable($location[The Dire Warren]))
```

Comment:
This isn't a good solution. I picked the Dire Warren as it's a no-wanderer zone and we have no reason to go there in a run so we shouldn't have anything override our Cleaver NC or our Cleaver NC override anything we actually want to hit.
I can't add a suggestion for the unchanged code so you'll need to change the section below which looks like:
```
return autoAdv($location[The Dire Warren]);
```
to be
```
location cleaverLocation = $location[The Dire Warren];
if (in_koe())
{
cleaverLocation = $location[Madness Bakery]; // arbitrarily picked always accessible location
}
return autoAdv($location[The Dire Warren]);
```
But like fix the formatting & make sure it validates.
Then the Cleaver adventures will still happen in KoE and not override other expected adventures in regular locations.

Reply:
cleaverLocation wasn't defined so I modified your code suggestion a bit to work.  it's running smoothly in KoE now, thanks MS